accounts/templates/dashboard.html

```diff
@@ -0,0 +1,9 @@
+<h3>
+    Witaj na stronie fiszkly, stronie,
+    ktora przyniesie nam duże korzyści finansowe.
+    Zapraszamy do korzystania. 
```

Comment:
Myślałem, że stronę robimy po angielsku? Nie wiem czy nawet przedmiot tego nie wymaga.
Bo lokalizacji i tak nie będziemy robić bo to za dużo roboty.

Reply:
Zmieniłem na język angielski. To było wpisane tylko testowo.